ani-cli

```diff
@@ -104,6 +106,17 @@ dep_ch() {
 
 # SCRAPING
 
+get_skip_times() {
+    skip_argument="$(ani-skip "$title" "$ep_no")"
+    echo "\"$title\"" "\"$ep_no\"" 1>&2
+    if [ "$skip_argument" = "" ]; then
+        printf "\033[1;31m%s\033[0m Failed\n" "ani-skip" 1>&2
+    else
+        printf "\033[1;32m%s\033[0m Skip Times Fetched\n" "ani-skip" 1>&2
+        skip_script="--script=~/.config/mpv/scripts/skip.lua"
+    fi
+}
```

Comment:
This function calling is not needed anymore.

Or we can just update it to exit ani-cli, if $skip_flag is null.

Reply:
Was completely removed.